apps/mobile/src/utils/navigation.ts

```diff
@@ -5,6 +5,29 @@ import {pipe} from 'fp-ts/function'
 import {safeParse} from './fpUtils'
 import * as O from 'fp-ts/Option'
 import reportError from './reportError'
+import {createNavigationContainerRef} from '@react-navigation/native'
+
+export const navigationRef = createNavigationContainerRef()
```

Comment:
This is the proper way how to handle navigation outside of react tree according to [official docs](https://reactnavigation.org/docs/navigating-without-navigation-prop/)